addons/web_editor/static/src/js/editor/snippets.options.js

```diff
@@ -4880,6 +4880,10 @@ registry.SnippetSave = SnippetOptionWidget.extend({
             dialog.on('closed', this, () => resolve());
         });
     },
+    // Enable the use of the save option only for snippet
+    _computeVisibility() {
+        return this.$target[0].hasAttribute("data-snippet");    
+    },
```

Comment:
- This function should not be in the "Options" area, it should be in the "Private" area.
- The function doc string is not correct.
- Your comment should finish with a dot

You should do:
```
//--------------------------------------------------------------------------
// Private
//--------------------------------------------------------------------------

/**
 * @override
 */
_computeVisibility() {
    // Enable the use of the save option only for snippets.
    return this.$target[0].hasAttribute("data-snippet");
},
```